developer/ruby/GenerateClass.rb

```diff
@@ -107,15 +122,30 @@ def forwardTranslate(options)
   end
 
   opts.on( '-i', '--iddObjectType IDDOBJECTTYPE', String, "IddObjectType to be wrapped by the ModelObject class being generated (ignored if not model sourceDirectory)") do |iddObjectType|
-    options [:iddObjectType] = iddObjectType
+    options[:iddObjectType] = iddObjectType
   end
 
   opts.on( '-r', '--[no-]reverseTanslator', "Autogenerate the ReverseTranslator code for IddObjectType") do |rt|
-    options [:reverseTranslator] = rt
+    options[:reverseTranslator] = rt
   end
 
   opts.on( '-f', '--[no-]forwardTanslator', "Autogenerate the ReverseTranslator code for IddObjectType") do |ft|
-    options [:forwardTranslator] = ft
+    options[:forwardTranslator] = ft
+  end
+
+  opts.on( '--clang-format-exe CLANGFORMAT', "Path to the clang-format executable") do |exe|
+    options[:clangFormatExe] = exe
+  end
+
+  opts.on_tail('-h', '--help', 'Print this help') do
+    puts opts.help
+    root_dir = File.expand_path("../../", File.dirname(__FILE__))
+    puts "\nExample Usage:"
+    puts '```'
+    puts "  $ cd developer/ruby"
+    puts '  $ /path/to/build/Products/openstudio GenerateClass.rb -c "SolarCollectorPerformancePhotovoltaicThermalBIPVT" -b "ModelObject"  -i "OS:SolarCollectorPerformance:PhotovoltaicThermal:BIPVT" -s model -o ' + File.join(root_dir, "src/model/") + ' -p -f -r'
+    puts '```'
+    return nil
```

Comment:
You can pass the specific clang-format executable if you want. And the help is cleaned up AND includes an example with a machine-specific path to the model/ folder